### PR TITLE
[JENKINS-34629] jenkins-ui (NPM module) is private, used only internally

### PR DESCRIPTION
**Pull request by recena**
[JENKINS-34629](https://issues.jenkins-ci.org/browse/JENKINS-34629)

Result of `frontend-maven-plugin:1.0:npm` after this PR is:

```
[INFO] Scanning for projects...
[INFO]                                                                         
[INFO] ------------------------------------------------------------------------
[INFO] Building Jenkins war 2.2-SNAPSHOT
[INFO] ------------------------------------------------------------------------
[INFO] 
[INFO] --- frontend-maven-plugin:1.0:npm (default-cli) @ jenkins-war ---
[INFO] Running 'npm install' in /Users/recena/Development/projects/jenkins/war
[INFO] ------------------------------------------------------------------------
[INFO] BUILD SUCCESS
[INFO] ------------------------------------------------------------------------
[INFO] Total time: 2.474 s
[INFO] Finished at: 2016-05-05T22:07:37+02:00
[INFO] Final Memory: 10M/309M
[INFO] ------------------------------------------------------------------------

Process finished with exit code 0
```

@reviewbybees especially @tfennelly

### CODE REVIEW

File: war/package.json [hunk 1]
```diff
@@ -3,6 +3,8 @@
   "version": "1.0.0",
   "description": "Jenkins User Interface",
   "license": "MIT",
+  "author": "Tom Fennelly <tom.fennelly@gmail.com> (https://github.com/tfennelly)",
```
Comment (recena) [inline war/package.json]: @tfennelly I've used the same information than you use in other NPM packages.

File: war/package.json [hunk 2]
```diff
@@ -3,6 +3,8 @@
...
+  "private": true,
```
Comment (recena) [inline war/package.json]: https://docs.npmjs.com/files/package.json#private

Comment (recena) [inline war/package.json]: This package is used only internally.

File: war/package.json (hunk 1 above)
Comment (oleg-nenashev) [inline war/package.json]: I would rather prefer something like "Jenkins contributors". NIT

Comment (recena) [inline war/package.json]: I can remove this line, it is unnecessary. I used it because @tfennelly designed everything around `js-modules`.

Comment (oleg-nenashev) [inline war/package.json]: I'm fine with it

**Comment by oleg-nenashev** [comment]
👍

**Comment by batmat** [comment]
:+1:

**Comment by recena** [comment]
@oleg-nenashev @batmat Thanks!

**Pull request merged**